design/v6-bundle/project/styles_v5.css

```
@@ -0,0 +1,32 @@
+/* v5 additions — stacked on top of v3 base */
+@import url('styles_v3.css');
```

Stylelint (17.10.0)

[error] Expected "url('styles_v3.css')" to be "'styles_v3.css'" (import-notation)

(import-notation)